librz/core/linux_heap_jemalloc.c

```diff
@@ -82,14 +82,14 @@ static bool GH(rz_resolve_jemalloc)(RzCore *core, char *symname, ut64 *symbol) {
 		}
 	}
 	if (!jemalloc_ver_end) {
-		eprintf("Warning: Is jemalloc mapped in memory? (see dm command)\n");
+		RZ_LOG_WARN("Warning: Is jemalloc mapped in memory? (see dm command)\n");
```

Comment:
```suggestion
		RZ_LOG_WARN("Is jemalloc mapped in memory? (see dm command)\n");
```
no need to write warning, since it is added by RZ_LOG_WARN

Reply:
@botta633 please apply this also on the one belows.

Reply:
Ye I will now!

Reply:
Please check my merge request

Reply:
or no need, I already modified the code. Thanks!